readme - making download_dependencies scripts.txt

```diff
@@ -0,0 +1,12 @@
+(Re)building buildUpdate.sh requires ruby and https://github.com/chrisvire/BuildUpdate
+Here's the command line commands I used:
+
+cd <path to where you want to generate the update scripts>
+<your path to buildupdate.rb (part of BuildUpdate repo above)>\buildupdate.rb -t bt321 -f download_dependencies_windows.sh
+<your path to buildupdate.rb (part of BuildUpdate repo above)>\buildupdate.rb -t bt435 -f download_dependencies_linux.sh
```

Comment:
Now that the files exist, `buildupdate.rb -f <filename>` will automatically read the `bt` from the file, so there is no need to specify other options.